src/components/interaction-designer/Block.vue

```diff
@@ -131,87 +132,80 @@
           @mouseenter="isConnectionSourceRelocateActive && activateExitAsDropZone($event, exit)"
           @mouseleave="isConnectionSourceRelocateActive && deactivateExitAsDropZone($event, exit)">
           <div v-if="!(exit.default && block.vendor_metadata.io_viamo.noValidResponse === NoValidResponseHandler.END_CALL)">
-          <div class="total-label-container">
-            <span class="badge badge-primary tree-block-item-label tree-block-item-output-subscribers-1" />
-          </div>
+            <div class="total-label-container">
+              <span class="badge badge-primary tree-block-item-label tree-block-item-output-subscribers-1" />
+            </div>
 
-          <div
-            class="block-exit-tag badge badge-warning"
-            @mouseenter="exitMouseEnter(exit)"
-            @mouseleave="exitMouseLeave(exit)">
-            <span class="block-exit-tag-text align-self-center"
-                  v-b-tooltip.hover.top="exit.test">
-              {{exit.name || '(untitled)'}}
-            </span>
-
-            <span
-              class="align-self-center">
-              <template v-if="exit.destination_block == null">
-                <plain-draggable
-                  :id="`exit/${exit.uuid}/pseudo-block-handle`"
-                  :key="`exit/${exit.uuid}/pseudo-block-handle`"
-                  v-b-tooltip.hover.top="transIf(isEditable, 'flow-builder.tooltip-new-connection')"
-                  class="btn btn-xs"
-                  :is-editable="isEditable"
-                  @initialized="handleDraggableInitializedFor(exit, $event)"
-                  @dragStarted="onCreateExitDragStarted($event, exit)"
-                  @dragged="onCreateExitDragged($event)"
-                  @dragEnded="onCreateExitDragEnded($event, exit)"
-                  @destroyed="handleDraggableDestroyedFor(exit)">
-                  <i
-                    v-if="isEditable && exitHovers[exit.uuid]"
-                    class="glyphicon glyphicon-move" />
-                  <font-awesome-icon
-                    v-else
-                    class="text-danger"
-                    :icon="['far', 'dot-circle']" />
-                </plain-draggable>
-
-                <template v-if="isConnectionCreateActive && isExitActivatedForCreate(exit) && livePosition">
+            <div
+              class="block-exit-tag badge badge-warning d-flex justify-content-center"
```

Comment:
Can we rename `block-exit-tag` to `block-exit-name` ? 🏷️

Reply:
Done!